### PR TITLE
minor fixes to the product attribution page

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-261

Description of Changes:
* Replace screenshot with codepen and update text
* Convert URL into a hyperlink

DOD:
- [x] nav.yml has been updated if applicable
- [x] file has been included where required if applicable
- [x] files removed have been deleted, not just excluded from the build if applicable

Review
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: general-configuration-guide/attribution-requirements.md [hunk 1]
```diff
@@ -12,14 +12,14 @@ There is a **POWERED BY TINY** attribution in the status bar that is enabled by
...
 * "**POWERED BY TINY**" to be stated or the [{{site.companyname}} logo]({{site.url}}/guidelines/#logo) placed in a readily visible location whenever the editor is loaded.
-* The word "**Tiny**" or the {{site.companyname}} logo should have a hyperlink to {{site.url}}.
+* The word "**Tiny**" or the {{site.companyname}} logo should have a hyperlink to [{{site.url}}]({{site.url}}).
 * The location of the product attribution is to be:
 
   * In the right-hand corner of the status bar of the {{site.productname}} editor; or
   * An alternate location adjacent to the {{site.productname}} editor that is no less prominent.
 
-The editor demo on the [{{site.companyname}} homepage]({{site.url}}) displays the status bar product attribution as in the screenshot below:
+For example, the following example shows the status bar product attribution in the bottom, right-hand corner:
```
Comment (lnewson) [inline general-configuration-guide/attribution-requirements.md:21]: Sounds a little weird having `example` repeated twice here, but I can't think of a better way to reword it :shrug:

File: general-configuration-guide/attribution-requirements.md [hunk 2]
```diff
@@ -12,14 +12,14 @@ There is a **POWERED BY TINY** attribution in the status bar that is enabled by
...
 
-![Demo]({{site.baseurl}}/images/screenshot-tiny-editor.png)
+{% include codepen.html id="basic-example" %}
```
Comment (lnewson) [inline general-configuration-guide/attribution-requirements.md:23]: Do we really want a codepen here? That could get confusing given it has a lot more functionality, etc...

File: general-configuration-guide/attribution-requirements.md (hunk 1 above)
Comment (EkimChau) [reply general-configuration-guide/attribution-requirements.md:21]: just remove 'For example, '?

Comment (EkimChau) [reply general-configuration-guide/attribution-requirements.md:21]: as an example

Comment (tylerkelly13) [reply general-configuration-guide/attribution-requirements.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1503/commits/96d49d5bb465257353154955fc3d11871afac5b6

File: general-configuration-guide/attribution-requirements.md (hunk 2 above)
Comment (tylerkelly13) [reply general-configuration-guide/attribution-requirements.md]: Swapped out this demo for another. Now using our most basic codepen. Note the original image was T4 and 'noisy'.

Comment (tylerkelly13) [reply general-configuration-guide/attribution-requirements.md]: after internal discussion, removed codepen, added an updated screenshot

**Pull request merged**